scheduler/task.go

```diff
@@ -35,11 +35,13 @@ import (
 	"github.com/intelsdi-x/snap/scheduler/wmap"
 )
 
+// const defines constants
```

Comment:
A description of what the DefaultDeadlineDuration and DefaultStopOnFailure do should be provided.